phase/generator.go

```diff
@@ -176,6 +197,19 @@ func (g *Generator) copyDockerfiles(dockerfiles []buildpack.DockerfileInfo) erro
 	return nil
 }
 
+func (g *Generator) copyContexts(contexts []buildpack.ContextInfo) error {
```

Comment:
Is this also not a no-op? I thought that was the point of providing `filepath.Join(g.GeneratedDir, launch.EscapeID(contextDir.ExtensionID))` as the `CNB_OUTPUT_DIR`